dv/uvm/core_ibex/Makefile

```diff
@@ -248,8 +244,24 @@ instr-gen-build-vars-prereq = \
 # Finally, $(instr-gen-build-vars-prereq) becomes a dependency of our target.
 ################## END EXAMPLE ###################
 
+core-config-vars-path := $(BUILD-DIR)/.cc.vars.mk
+-include $(core-config-vars-path)
+
+core-config-var-prereq = $(call vars-prereq,gen,Generate core configuration file,$(core-config-var-deps))
+$(CORE-CONFIG-STAMP): \
+	$(core-config-var-prereq) ./riscv_dv_extension/riscv_core_setting.tpl.sv \
```

Comment:
We've been using the convention of two spaces for wrapping lines of a rule's prerequisites in this file. Could you tweak this to match?